lint.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```

Comment:
This gets moved to `linux/` in the first commit, then moved back again soon after - do you mind dropping that intermediate move? That would make it easier to compare what changed.

It looks like it later gets moved to ci/lint.sh too, if you could combine that as well it would be helpful.

Reply:
> This gets moved to linux/ in the first commit, then moved back again soon after - do you mind dropping that intermediate move? That would make it easier to compare what changed.

Will do!

> It looks like it later gets moved to ci/lint.sh too, if you could combine that as well it would be helpful.

Hmm? I don't see that?